src/components/UserManagement/UserBilling/UserBilling.tsx

```diff
@@ -48,12 +48,8 @@ export default class UserBilling extends React.Component<Props, State> {
         customer: value,
       });
     });
-    const response = await RESTRequest(
-      "GET",
-      "customers/me/stored-payment-methods"
-    );
-    const methods = await response.json();
-    this.setState({ paymentMethods: methods });
+    const storedPaymentMethods = this.context.storedPaymentMethods;
+    this.setState({ paymentMethods: storedPaymentMethods });
```

Comment:
Do we still need to keep the methods on the state? We should be able to display them directly from the context. This approach has the problem that if you refresh the page while on the billing tab, the cards won't be there, as by the time the component mounts, the user wasn't fetched yet.